legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -475,8 +475,14 @@ def alter_outputs(filing: FilingStorage, outputs: set):
         if filing.filing_type == 'alteration':
             if filing.meta_data.get('alteration', {}).get('toLegalName'):
                 outputs.add('certificateOfNameChange')
-        elif filing.filing_type == 'specialResolution' and 'changeOfName' in filing.meta_data.get('legalFilings', []):
-            outputs.add('certificateOfNameChange')
+        elif filing.filing_type == 'specialResolution':
+            if 'changeOfName' in filing.meta_data.get('legalFilings', []):
+                outputs.add('certificateOfNameChange')
+            if 'alteration' in filing.meta_data.get('legalFilings', []):
+                if filing.filing_json['filing']['alteration'].get('memorandumInResolution') is True:
+                    outputs.remove('certifiedMemorandum')
+                if filing.filing_json['filing']['alteration'].get('rulesInResolution') is True:
+                    outputs.remove('certifiedRules')
```

Comment:
so for 'memorandumInResolution' and 'rulesInResolution', are you planning on updating the `alteration` schema?

Also, there is already a [rulesChangeInSR](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/alteration.json#L57-L60) that Ritvick had added to the alteration schema awhile back.

Reply:
Yup, that's the plan - I'll update the other my other PR to reflect it.

Reply:
Are there any legal api validation updates required for SRs to validate some of these new scenarios?

Reply:
It's in there now

Reply:
awesome, thanks